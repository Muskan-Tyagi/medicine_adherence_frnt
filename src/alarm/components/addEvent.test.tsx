import React from 'react';
import renderer from 'react-test-renderer';
import Addevent from './addEvents'

describe('Add events', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(< Addevent/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});