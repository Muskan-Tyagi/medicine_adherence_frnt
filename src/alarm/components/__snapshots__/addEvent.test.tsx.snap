// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add events renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;

exports[`Home Screen renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "height": "100%",
      "width": "100%",
    }
  }
/>
`;
